ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -25,26 +24,26 @@
     private String reason;
     private String feeCode;
     private String feeAccountNumber;
-    private BigInteger feeAmountInPennies;
+    private String feeAmountInPennies;
```

Comment:
The feeAmountInPennies was in BigInteger and it was `MoneyGBP` in CCD. Similar is what we should do rather than changing to String

Reply:
As per CCD Data formats:
```
A MoneyGBP value is a string representing a monetary value in pence.
```

Reply:
`feeAmountInPennies ` was being displayed as pennies and was not MoneyGBP before.

Reply:
The bigInteger will be converted to String as well. It is working already

Reply:
If `BigInteger` is recommendation from CCD team, I am happy to change that. Their recommendation in docs says its `String`.